src/delete-branch.ts

```diff
@@ -1,4 +1,4 @@
-import { deleteBranch } from "./lib/crowdin";
+import { deleteBranch, deleteTask, listTasks } from "./lib/crowdin";
 import log from "./utils/logging";
 
 interface DeleteBranchOptions {
```

Comment:
Should we conditionally delete tasks depending on an input option? I can go either way on this.

Btw, I think I'm missing a test for the task deletion part 🙂